### DISCUSSION TITLE
API: How to reblog a post from another instance?

### DISCUSSION THREAD

**Discussion by tmladek** [mastodon/mastodon#20409] [Q&A] [answered]
The *Boost* section in the docs at https://docs.joinmastodon.org/methods/statuses/ says:

>Path Parameters
>`:id` - Local ID of a status in the database.

But how do I find out the "Local ID" for a post that I only have an URL of?

If I got the post via the `timelines` endpoint, I suppose that I could just use the `id` from there - but what about posts not on my timeline, that have been posted on a different instance?

Thanks in advance!

**Comment by trwnh** [comment, marked as answer]
Use the search api with the resolve parameter and q = the URL
